applications/nrf5340_audio/bin/manifest.yaml

```diff
@@ -0,0 +1,7 @@
+description: Packetcraft Controller
```

Comment:
Shall we use the name "LE Audio Controller Subsystem for nRF53" which shown in the doc?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.9.99-dev1/nrf/applications/nrf5340_audio/README.html#legal-notices-and-disclaimers

Reply:
To my understanding "LE Audio Controller Subsystem for nRF53" is the name we should use. This name is consistently used in the documentation.

Reply:
Updated